app/workers/scheduler/self_destruct_scheduler.rb

```diff
@@ -20,7 +20,7 @@ def perform
   private
 
   def sidekiq_overwhelmed?
-    redis_mem_info = Sidekiq.redis_info
+    redis_mem_info = Sidekiq.default_configuration.redis_info
```

Comment:
Is this going to hold the proper information, given that Redis is configured by the `Sidekiq.configure_server` and `Sidekiq.configure_client` blocks?